src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -1979,6 +1984,27 @@ def SetCheckMatchingFabric(self, check: bool):
             lambda: self._dmLib.pychip_DeviceController_SetCheckMatchingFabric(check)
         ).raise_on_error()
 
+    def SetIcdRegistrationParameters(self, symmetricKey: bytes = None, checkInNodeId: int = 0, monitoredSubject: int = 0, stayActiveMs: int = 0):
```

Comment:
would it make maybe sense to have a `@dataclass` for parameters instead of several values? Or mark these as `**, symmetricKey: bytes = None, checkInNodeId: int = 0, monitoredSubject: int = 0, stayActiveMs: int = 0` to force to always name these arguments?

would a `stayActiveMs` being a timeDelta make sense?
A comment should maybe explain what a 0 means for stayActiveMs.

Reply:
I would prefer stayActiveMs or StayActiveDurationMs that could explicitly know the functionality about this parameteer which is to send stayActive request in the end of commissioning flow.

Reply:
Makes sense

I feel `SetIcdRegistrationParameters` which can accept 0 arguemtns a bit weird. So I renamed it to EnableICDRegistration